public/js/signup.js

```diff
@@ -8,14 +8,14 @@ const signupFormHandler = async (event) => {
     const state = document.querySelector ('#state-signup').value.trim();
 
     if (firstName && lastName && email && password && state) {
-        const response = await fetch ('/api/users', {
+        const response = await fetch ('/api/users/create', {
             method: 'POST',
             body: JSON.stringify ({firstName, lastName, email, password, state}),
             headers: {'Content-type': 'application/json'}
         });
 
         if (response.ok) {
```

Comment:
As part of the logout, this page will be redirected to the signin page: 
![image](https://user-images.githubusercontent.com/9697002/204113405-37dfe863-f74f-49be-b9ea-033941894faa.png)

Reply:
as per our conversation, this comment is incorrect